thoth/python/artifact.py

```diff
@@ -71,15 +71,22 @@ def _extract_py_module(self) -> None:
         try:
             self.dir_name = tempfile.mkdtemp()
             try:
-                with zipfile.ZipFile(self.compressed_file) as zip_ref:
-                    zip_ref.extractall(self.dir_name)
-                    _LOGGER.debug("Artifact is .whl")
+                if self.compressed_file.endswith('.tar.gz'):
+                    tf = tarfile.open(self.compressed_file)
+                    tf.extractall(self.dir_name)
+                    _LOGGER.debug("Artifact is .tar.gz file")
+                else:
+                    ext = self.compressed_file.split('.')[-1]
+                    if ext in ['.msi', 'rpm']:
```

Comment:
@fridex looks like you already approved, :smile: ! But, I wanted to add some logic like this to skip those files.

Reply:
Oh let me just revert back, deleting last commit